js/gated-content/src/views/LiveStreamPage.vue

```diff
@@ -201,17 +202,24 @@ export default {
           this.video = this.combine(response.data.data, response.data.included, this.params);
           this.multipleReferencesWorkaround(response);
           this.loading = false;
+
+          const currentDate = new Date();
+          const startDate = new Date(this.video.attributes.date.value);
```

Comment:
we should try to use dayjs everywhere to fix possible bugs in different timezones